x-pack/plugins/maps/public/classes/styles/vector/properties/dynamic_color_property.tsx

```diff
@@ -145,81 +145,59 @@ export class DynamicColorProperty extends DynamicStyleProperty<ColorDynamicOptio
   }
 
   _getOrdinalColorMbExpression() {
-    const targetName = this.getFieldName();
+    let colorStops: Array<number | string> | null = null;
     if (this._options.useCustomColorRamp) {
       if (!this._options.customColorRamp || !this._options.customColorRamp.length) {
         // custom color ramp config is not complete
         return null;
       }
-
-      const colorStops: Array<number | string> = this._options.customColorRamp.reduce(
-        (accumulatedStops: Array<number | string>, nextStop: OrdinalColorStop) => {
-          return [...accumulatedStops, nextStop.stop, nextStop.color];
-        },
-        []
-      );
-      const firstStopValue = colorStops[0] as number;
-      const lessThanFirstStopValue = firstStopValue - 1;
-      return [
-        'step',
-        ['coalesce', [this.getMbLookupFunction(), targetName], lessThanFirstStopValue],
```

Comment:
With this PR,  "makeMbClampedNumberExpression" is now used for clamping values below the first stop to the first stop. This has the added benefit of users not having to specify the low range of there color ramp.

Reply:
discussed offline. This makes the `custom color ramp` editor for ordinal scales a little more confusing, since the first stop value is never reflected back in the legend. Disabling the stop-input of the first color should make this linking more intuitive.